webpack/ForemanTasks/Components/TasksDashboard/Components/TasksCardsGrid/Components/StoppedTasksCard/StoppedTasksCard.js

```diff
@@ -26,6 +27,8 @@ const StoppedTasksCard = ({
   ...props
 }) => {
   const { STOPPED } = TASKS_DASHBOARD_AVAILABLE_QUERY_STATES;
+  const otherCount = data.other || 0;
+  delete data.other;
```

Comment:
IMO it would be better to send the data correctly from the server, instead of manipulating it here.

Reply:
What do you mean by getting it correctly? I can move the default `0` to the selector/ default props

Reply:
the `data` object looks something like:
`data = { error, success, warning, other }`
instead of manipulating it here and later also using `calcStoppedOther` helper,
we could return from the server something like:
`data = { results: { error, success, warning }, otherCount: 0 }`